### PR TITLE
Feature/shell write

### PR DESCRIPTION
**Pull request by zaeng616**
## 📌 PR 제목
- [변경 유형] Test Shell의 write 기능 구현

## 📄 변경 사항
- SSD Interface의 write 를 이용하여 write 하는 기능 구현

## 🔍 상세 설명
- 기본 write 기능과 edge case 추가 구현하였습니다.

## ✅ 체크리스트
- [x] 코드 스타일을 따랐는가?
- [x] 테스트를 작성했는가?
- [x] master branch 리베이스 후 빌드 확인 하였는가?

### CODE REVIEW

File: TestShell/test_shell.cpp [hunk 1]
```diff
@@ -4,4 +4,12 @@ using std::cout;
 
 void TestShell::read(int lba) {
 	cout << ssd->read(lba);
-}
\ No newline at end of file
+}
+
+void TestShell::write(int lba, std::string value) {
+	if (ssd == nullptr) return;
```
Comment (dooyeun) [inline TestShell/test_shell.cpp:10]: 다른 feature 들도 구현되고 나면 유효성 검사는 따로 빼는 것도 좋아보입니다.

**Review by dooyeun — APPROVED** [review]
Unit Test들도 Fixture로 구현하여 여러가지 테스트 케이스들을 만들어주셨습니다.
모든 feature들이  구현된 후 refactoring 해야할 사항을 기록용으로 남겨두었습니다.
고생하셨습니다.

**Review by doky0130 — COMMENTED** [review]
unit test를 다양하게 구현하셨네요 저도 read unit test를 비슷하게 추가해야겠어요!

**Review by doky0130 — APPROVED** [review]
